src/audio/pipeline.c

```diff
@@ -694,12 +701,14 @@ static void pipeline_schedule_triggered(struct pipeline_walk_context *ctx,
 	struct list_item *tlist;
 	struct pipeline *p;
 	uint32_t flags;
+	int i = 1;
 
 	irq_local_disable(flags);
 
 	list_for_item(tlist, &ctx->pipelines) {
 		p = container_of(tlist, struct pipeline, list);
-
+		printf("pipeline_schedule_triggered: pipeline number %d\n", i);
```

Comment:
pipeline_schedule_triggered() is the first function to call pipeline_schedule_copy(p,0). Read from the code this is a for-loop iterating all pipeline in the list. And after adding the debug code, I found there are 2 pipeline iteration (both pipeline should have called pipeline_schedule_copy(p,0)).

Output log:
https://paste.googleplex.com/6407279421161472
pipeline_schedule_triggered: pipeline number 1
unknown pipeline_schedule_copy
pipeline_schedule_triggered: pipeline number 2
unknown pipeline_schedule_copy